frontend/src/queries/schema.ts

```diff
@@ -1149,6 +1149,7 @@ export interface QueryRequest {
      * see the [PostHog HogQL documentation](/docs/hogql#api-access).
      */
     query: QuerySchema
+    filters_override?: DashboardFilter
```

Comment:
This is not actually an override, since it adds to the existing filters, not replace. How about literally `dashboard_filters`?

Reply:
Should I change this just for `/query` endpoint?
As for `/insight` we already have logic to read `filters_override` passed down from the `/dashboard` endpoint [here](https://github.com/PostHog/posthog/blob/master/posthog/api/insight.py#L560), seems a bit unnecessary to read both `filters_override` for dashboard filters and `dashboard_filters` when loading an insight.

Reply:
another option is to rename all of it to `dashboard_filters_override` or something, can address that in a separate PR as this will become a lot bigger then :)

Reply:
Oh, didn't see the pre-existing `filters_override`, in that case does seem simplest to just reuse that